packages/components/src/gradient-picker/index.tsx

```diff
@@ -107,25 +111,72 @@ function MultipleOrigin( {
 	);
 }
 
-export default function GradientPicker( {
+function Component( props: PickerProps< any > ) {
+	if ( isMultipleOrigin( props.gradients ) ) {
+		return <MultipleOrigin { ...props } />;
+	}
+	return <SingleOrigin { ...props } />;
+}
```

Comment:
This will replace the the `Component` declaration via a ternary a little later in the file.